extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -236,6 +244,24 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     super.onCreativeRender(isVerifiedAmpCreative);
     injectActiveViewAmpAnalyticsElement(
       this, this.extensions_, this.ampAnalyticsConfig);
+    this.centerCreative_();
```

Comment:
Since this is AdSense specific, would we want to put this change in amp-a4a? Or did you have somewhere else in mind?